test/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+contracts=(
+    "ZombieAttack"
+)
+
+truffle compile || exit 1
+for contractName in "${contracts[@]}"
```

Comment:
We are only using these for testing, right? If so, let's move the compiled files from the root level `artifacts/` and into `test/artifacts/`.

Reply:
doe the same go for `near_evm.wasm`?

Reply:
I think so? Do we only use that for testing? If so, then just move that entire directory